app/javascript/components/workers-form/helpers.jsx

```diff
@@ -0,0 +1,237 @@
+/**
```

Comment:
I feel like we shouldn't be writing (most of) this ourselves.  There has got to be a package that does presentation of bytes.  A quick google showed me https://www.npmjs.com/package/pretty-bytes, but maybe lodash has something?

Reply:
I understand, there are a lot of options, but I didn't want to add another dependency for such simple function.

Reply:
@rvsia i agree if we need to add similar calculations/functionality on any other screen we can look for a package in future but for this PR we can go with what you have now.